rhizome/victoriametrics/bin/configure

```diff
@@ -0,0 +1,95 @@
+#!/bin/env ruby
```

Comment:
It seems in some files we use `victoria_metrics` and in some others we use `victoriametrics`. It would be good to be consistent about them.